__tests__/round.test..js

```diff
@@ -0,0 +1,114 @@
+import Round from '../app/model/round.js';
```

Comment:
omg. the number of times i fixed this then didn't check it in cuz it as too small of a change. t hen i `git reset --hard`. blargh.